layer/Python/Dockerfile

```diff
@@ -1,19 +1,26 @@
 FROM public.ecr.aws/lambda/python:3.8
 
-
-
 ARG PACKAGE_SUFFIX=''
 
 USER root
 WORKDIR /tmp
 
-
-# PACKAGE_SUFFIX = '[pydantic]==1.23.0'
-# PACKAGE_SUFFIX = '[pydantic]'
-# PACKAGE_SUFFIX = '=='1.23.0'
+# PACKAGE_SUFFIX = '[all]==2.0.0'
+# PACKAGE_SUFFIX = '[all]'
+# PACKAGE_SUFFIX = '=='2.0.0'
 # PACKAGE_SUFFIX = ''
 
+RUN yum update -y && yum install -y zip unzip wget tar gzip binutils
 
-RUN yum update -y && yum install -y zip unzip wget tar gzip
+RUN pip install -t /asset/python aws-lambda-powertools$PACKAGE_SUFFIX
 
-RUN pip install -t /asset/python aws-lambda-powertools$PACKAGE_SUFFIX
\ No newline at end of file
+# Removing nonessential files
```

Comment:
We can also remove dependencies boto3 brings, namely: `s3transfer`, `python-dateutil`, `urllib3`. We don't want to remove `jmespath` because we're installing it on L15.

* Boto3 deps: https://github.com/boto/boto3/blob/develop/requirements.txt
* Botocore deps: https://github.com/boto/botocore/blob/develop/setup.py#L25

---

```shell
pip install boto3 -t /tmp/boto
rm -rf /tmp/boto/boto*
```

Listing remaining files - Nothing substantial per se, but for the sake of completeness:

```shell
ls -lh /tmp/boto

drwxr-xr-x   3 lessa  staff    96B Oct  3 15:58 __pycache__
drwxr-xr-x   4 lessa  staff   128B Oct  3 15:58 bin
drwxr-xr-x  14 lessa  staff   448B Oct  3 15:58 dateutil
drwxr-xr-x  11 lessa  staff   352B Oct  3 15:58 jmespath
drwxr-xr-x   8 lessa  staff   256B Oct  3 15:58 jmespath-1.0.1.dist-info
drwxr-xr-x   9 lessa  staff   288B Oct  3 15:58 python_dateutil-2.8.2.dist-info
drwxr-xr-x  19 lessa  staff   608B Oct  3 15:58 s3transfer
drwxr-xr-x   9 lessa  staff   288B Oct  3 15:58 s3transfer-0.6.0.dist-info
drwxr-xr-x   8 lessa  staff   256B Oct  3 15:58 six-1.16.0.dist-info
-rw-r--r--   1 lessa  staff    34K Oct  3 15:58 six.py
drwxr-xr-x  17 lessa  staff   544B Oct  3 15:58 urllib3
drwxr-xr-x   8 lessa  staff   256B Oct  3 15:58 urllib3-1.26.12.dist-info
```